src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -326,10 +326,20 @@ public function prepare(Request $request): static
     /**
      * Sends HTTP headers.
      *
+     * @param null|positive-int $statusCode The status code to use. Override the statusCode property if set and not null.
+     *
      * @return $this
      */
     public function sendHeaders(): static
```

Comment:
I believe we usually comment out the param
```suggestion
    public function sendHeaders(/* ?int $statusCode = null */): static
```